### PR TITLE
Add kubeflow on aws docs

### PR DESCRIPTION
**Pull request by Jeffwan**
AWS recently add some aws specific features to kubeflow and we also like to have these docs to help users understand features. 

Address doc issue.
https://github.com/kubeflow/website/issues/621

We have a static server here. Links in the main page seems doesn't work because of hugo config issue, I use a nginx to proxy traffic. Please click link on left sidebar to enter page. 
http://54.213.19.198/docs/aws/

Code changes for aws:
https://github.com/kubeflow/kubeflow/pull/2823
https://github.com/kubeflow/kubeflow/pull/2983

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/657)

### CODE REVIEW

**Comment by Jeffwan** [comment]
Does the bot automatically recheck CLA? @nrdlngr already signed CLA for himself.

**Comment by nrdlngr** [comment]
Approved.

**Comment by sarahmaddox** [comment]
@Jeffwan We have Netlify running, which means you get a nice preview when you submit a docs PR.

The preview for this PR is here:
https://deploy-preview-657--competent-brattain-de2d6d.netlify.com/docs/aws/

And here:
https://deploy-preview-657--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-aws/

File: content/docs/aws/authentication.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+weight = 90
++++
+
+This section shows the how to add TLS support and create a user pool to authenticate users with your custom domain.
+
+
+## Traffic Flow
+External Traffic → [ Ingress → Istio ingress gateway → ambassador ]
+
+When you generate and apply kubernetes resources, an ingress is created to manage external traffic to Kubernetes services. The AWS ALB Ingress Controller will provision an Application Load balancer for that ingress. By default, TLS and authentication are not enabled at creation time.
```
Comment (sarahmaddox) [inline content/docs/aws/authentication.md]: Please expand or explaing "ALB" on first usage.

File: content/docs/aws/deploy/install-kubeflow.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        istio-system   istio-ingress   *       a743484b-istiosystem-istio-2af2-xxxxxx.us-west-2.elb.amazonaws.com   80      1h
+        ```
+
+        This deployment may take 3-5 minutes to become ready. Verify that the address works by opening it in your preferred Internet browser. You can also run `kubectl delete istio-ingress -n istio-system` to remove the load balancer entirely.
+
+    * If you didn't create a load balancer, please use port-forwarding to visit your cluster. Run following command and visit `localhost:8080`.
+
+        ```shell
+        kubectl port-forward -n kubeflow `kubectl get pods -n kubeflow --selector=service=ambassador -o jsonpath='{.items[0].metadata.name}'` 8080:80
+        ```
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/aws/deploy/install-kubeflow.md]: My above comment is probably useful on other pages in this PR too.

File: content/docs/aws/features.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
++++
+title = "Kubeflow on AWS Features"
+weight = 200
++++
+
+## Reasons to use Kubeflow on AWS
+
+Running Kubeflow on Amazon EKS brings the following optional and configurable features:
+
+* You can manage your Amazon EKS cluster provisioning with **eksctl** and easily choose between multiple compute and GPU worker node configurations.
```
Comment (sarahmaddox) [inline content/docs/aws/features.md]: Suggestion: Add a link for each of the features on this page, so people who don't know what they are can easily find more info.

File: content/docs/aws/deploy/existing-cluster.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
++++
+title = "Init cluster setup for existing cluster"
```
Comment (sarahmaddox) [inline content/docs/aws/deploy/existing-cluster.md]: s/Init/Initial

**Comment by sarahmaddox** [comment]
@googlebot please re-check CLAs. All contributors have signed and approved this PR.

File: content/docs/aws/deploy/install-kubeflow.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
++++
+title = "Install Kubeflow"
+description = "Instructions for deploying Kubeflow with the shell"
+weight = 4
++++
+
+This guide describes how to use the `kfctl.sh` script to
+deploy Kubeflow on AWS.
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply content/docs/aws/deploy/install-kubeflow.md]: Good point. I will address it in the doc.

**Comment by Jeffwan** [comment]
@sarahmaddox  Thanks a lot for your review! I addressed all the feedbacks and update docs. Please have another look.

Another thing we can not clear is CLA issue.  @nrdlngr signed CLA and how could he verify author consent? We are not sure why command bot accepts and he left `Approved` here.

` CLAs are signed, but unable to verify author consent`

**Comment by jlewi** [comment]
To fix the CLA check; you need to rebase and change the commit authors so its a single author who matches PR author.

If you rebase and squash commits that will fix it.

**Comment by Jeffwan** [comment]
@sarahmaddox Double check bot message and notice `cla/google` won't change to `Yes`. Approval can manually update label or move forward directly. Since @nrdlngr already agree to contribute, I think we are good on CLA parts.

**Comment by sarahmaddox** [comment]
Thanks! Agreed we're good to go on the CLA. I'll approve this and see if the merge happens automatically, as we don't have the CLA label in this repo. If that doesn't work, I'll try merging myself.

/lgtm
/approve

**Pull request merged**